zest/releaser/utils.py

```diff
@@ -849,19 +849,32 @@ def get_last_tag(vcs, allow_missing=False):
     # by setuptools' version parsing. A direct match is obviously
     # right. The 'less' approach handles development eggs that have
     # already been switched back to development.
-    available_tags.reverse()
-    found = available_tags[0]
+    # Note: if parsing the current version fails, there is nothing we can do:
+    # there is no sane way of knowing which version is smaller than an unparsable
+    # version, so we just break hard.
     parsed_version = parse_version(version)
+    found = parsed_found = None
     for tag in available_tags:
-        parsed_tag = parse_version(tag)
-        parsed_found = parse_version(found)
+        try:
+            parsed_tag = parse_version(tag)
+        except Exception:
+            # I don't want to import this specific exception,
+            # because it sounds unstable:
+            # pkg_resources.extern.packaging.version.InvalidVersion
+            continue
```

Comment:
Would a `logger.debug(e)` be handy here for debugging if something goes wrong with versions?

Reply:
Done.